jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -189,6 +189,7 @@ class PlotOptions(PluginTemplateMixin):
     stretch_vmax_sync = Dict().tag(sync=True)
 
     stretch_hist_zoom_limits = Bool().tag(sync=True)
+    stretch_hist_nbins = Int().tag(sync=True)

```

Comment:
this would benefit from using `IntHandleEmpty` and making sure that the case of the empty widget is handled correctly in `set_histogram_bin_size`.